compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt

```diff
@@ -80,8 +80,6 @@ import platform.darwin.NSObject
 
 private val DUMMY_UI_ACCESSIBILITY_CONTAINER = NSObject()
 
-// TODO: Impl for UIKit interop views
```

Comment:
Is it solved? Should we mention it in the release notes?

Reply:
Yes, it was for 1.6.11, we forgot to remove TODO.